### ISSUE TITLE
RLV file doesn't work in sync with video and gpx any more

### ISSUE THREAD

**Issue by MKH-42** [GoldenCheetah/GoldenCheetah#3959]
Issue tracker is **only** for Bugs and Features, please don't open issues for questions or technical support. Before to open a new issue please read the contributing guidelines (link below).

When I use GC with an RLV and a gpx and a video it doesn't work any more in sync.
The speed doesn't linked to the video.

GC Version:
Build date: Jul 9 2021 14:55:51
Build id: 4005
OS: Windows 10

not with all but with some rlv files.

It works well with version from january.  (v3.6-DEV2101)

Effect: In the beginning you can drive as fast as possible but the video move only very slow.
It is not the video. I tried out the video with other combinations. It works.
So it should be the RLV or GPX.

[test.zip](https://github.com/GoldenCheetah/GoldenCheetah/files/6874273/test.zip)
Here the gpx and rlv file.

**Comment by amtriathlon** [comment]
Main change introduced to videosync after v3.6-DEV2101 was https://github.com/GoldenCheetah/GoldenCheetah/pull/3764, perhaps the original author or other participants there may help.

**Comment by MKH-42** [comment]
I think it is NOT the video sync. It is the RLV reader which doesn't work anymore.
JSON and video works good.
TTS and video works good.
RLV and video doesn't work in all cases. #3764 is an VLC handle change.

**Comment by amtriathlon** [comment]
RLV reader has not changed this year, see https://github.com/GoldenCheetah/GoldenCheetah/commits/master/src/Train/VideoSyncFile.cpp
